### PR TITLE
142 multiple choice server feedback

### PR DESCRIPTION
**Pull request by ysahnpark**
Minor changes to IpsAnswerMan to match the message structures passed back and forth with Ips/CorrecnessEngine.
Added logging here and there.

Also added Multiple Choice Question integration demo files.

@mlippert, @seannives your reviews are appreciated.

### CODE REVIEW

File: test/integration/mcq-int.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
+/*****************************************************************************
+ * Integration test for Submissions.
+ *
+ * This test is integration test and is meant to run while IPS server is up.
+ *
+ * If you are getting error in the Browser console saying: 
+ * "XMLHttpRequest cannot load http://localhost:8088/sequencenodes/. 
+ * Origin null is not allowed by Access-Control-Allow-Origin"
+ *
```
Comment (seannives) [inline test/integration/mcq-int.js:11]: If you get this error what does it mean?

File: test/integration/mcq-int.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+ * Any issue with the server, try refreshing the Redis cache
+ *
```
Comment (seannives) [inline test/integration/mcq-int.js:13]: Could you put the command to refresh the cache here too please?

Comment (ysahnpark) [inline test/integration/mcq-int.js:13]: Good point.

The mcq-int.js is client javascript, therefore no API to refresh cache from here.
For administrative purposes, some people exposes hidden REST API to refresh cache from remote.

We could think about that too.

**Comment by mlippert** [comment]
:+1:

**Pull request merged**